Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -233,8 +237,24 @@ private extension LayoutManager {
                     start = textStorage.numberOfItems(in: list, at: enclosingRange.location)
                 }
             }
+
+            var indentLevel = 1
+            // Determine indentation level, if needed. The indentation level is only used by the standard list style
+            if list.style == .unordered, listIndentStyle == .standard {
+                // only get the width of the first level once
+                if firstLevelWidth == nil {
+                    firstLevelWidth = paragraphStyle.indentToFirst(TextList.self)
+                }
+
+                // calculate current indent level
+                let indentWidth = paragraphStyle.indentToLast(TextList.self)
+                if let firstLevelWidth = firstLevelWidth {
```

Comment:
Nit: I'd consider refactoring this since we know that at this point `firstLevelWidth` should never be `nil`.

Reply:
Agreed